hooks/useClickOutside.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react';
+import type { RefObject, MouseEvent, TouchEvent } from 'react';
+
+// eslint-disable-next-line no-unused-vars
+type TypeHandler = (event: MouseEvent | TouchEvent) => void;
```

Comment:
```suggestion
type TypeHandler = (_event: MouseEvent | TouchEvent) => void;
```

Reply:
ESlint was not happy with this change 🤔
I'm searching deeply

Reply:
Probably Hinata's change wasn't merged yet to make eslint not complain about this.